test_configs/verbs/verbs.exclude

```diff
@@ -22,3 +22,7 @@ unexpected_msg
 rma.+rdm.+writedata
 rdm.+atomic
 rdm.+cntr
+inj_complete
```

Comment:
This test should work for verbs/RDM. If we remove FI_TAGGED, it might work for all EP types

Reply:
Yes. It works for RDM type. I'll update the PR. Thanks.